src/main/java/pl/multiplex/multiplexreservation/dao/ReservationRepo.java

```diff
@@ -1,4 +1,27 @@
 package pl.multiplex.multiplexreservation.dao;
 
-public interface ReservationRepo {
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+import pl.multiplex.multiplexreservation.dao.entity.Reservation;
+import pl.multiplex.multiplexreservation.dao.entity.Screening;
+import pl.multiplex.multiplexreservation.dao.entity.Seat;
+
+import java.util.List;
+
+@Repository
+@Transactional(readOnly = true)
```

Comment:
1. Co daje nam takie ustawienie transakcji?
2. Czy to jest najlepsze miejsce na ustawienie transakcji i dlaczego nie? :)

Reply:
W poradnikach i tutorialach jakie oglądałem, używali tego i nie zastanawiając się do czego to dokładnie służy, po prostu tego użyłem.